torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -476,31 +476,34 @@ class InsertObserversHelper {
   // nn.Linear + nn.ReLU
   const PatternInfo nn_linear_nn_relu = PatternInfo::parse_from_str(
       R"(
-graph(%self, %linear, %relu, %input):
+graph(%input, %linear, %relu):
```

Comment:
For my understanding, why was self part of the graph, when it is not used in the actual pattern?

Reply:
yeah, that's why I removed it, it's not needed in the pattern.